src/ReadSpeed.cxx

```diff
@@ -235,6 +235,11 @@ Result ReadSpeed::EvalThroughputMT(const Data &d, unsigned nThreads)
    const auto rangesPerFile = MergeClusters(GetClusters(d), maxTasksPerFile);
    clsw.Stop();
 
+   size_t nranges = 0;
+   for (const auto &r : rangesPerFile)
+      nranges += r.size();
+   std::cout << "Total number of entry ranges: " << nranges << '\n';
```

Comment:
I think a more user-oriented way to say this would be "Total number of tasks: " -- the interesting information is that the number of entry ranges represent the level of parallelism of this workload.